tests/wpt/web-platform-tests/html/semantics/forms/the-textarea-element/textarea-maxlength.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>textarea maxlength</title>
+  <link rel="author" title="tigercosmos" href="mailto:phy.tiger@gmail.com">
+  <link rel=help href="https://html.spec.whatwg.org/multipage/#attr-textarea-maxlength">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+
+<body>
+
+  <textarea id="none"></textarea>
+  <textarea id="negative" maxlength="-5"></textarea>
+  <textarea id="non-numeric" maxlength="not-a-number"></textarea>
+  <textarea id="assign-negative"></textarea>
```

Comment:
nit: You don't need these just to assign to it, right? You can use `document.createElement('textarea')` instead.

Reply:
We should probably also test that `setAttribute("maxlength")` changes the value reflected in `maxLength`, and vice versa.

Reply:
Actually these are textarea version modified from `html/semantics/forms/the-input-element/maxlength.html`.

About `setAttribute("maxlength")`, it has already tested well in ` tests/wpt/metadata/html/dom/reflection-forms.html`

Reply:
Do you think we should refactor the code or this is ok?

Reply:
Yeah, I guess this is fine.